api.raml

```diff
@@ -529,6 +552,12 @@ types:
 /mfa:
   post:
     description: Create a new mfa record.
+    queryParameters:
+      rpOrigin?:
+        type: string
+        description: The Relay Party origin url (only needed for WebAuthn)
+        default: ""
+        example: "https://sub.domain.com"
```

Comment:
Is this truly expected to be sent with the "https://", or is the receiving code just expecting a domain name?

Reply:
The RPID is only the `domain.com` portion, but Origin includes scheme